### PR TITLE
chore: upgrade lit to 2.0.0

### PR DESCRIPTION
**Pull request by haijian-vaadin**

### CODE REVIEW

File: frontend/packages/common-frontend/package.json [hunk 1]
```diff
@@ -45,7 +45,7 @@
     "access": "public"
   },
   "dependencies": {
-    "lit": "^2.0.0-rc.2"
+    "lit": "2.0.0"
```
Comment (Lodin) [inline frontend/packages/common-frontend/package.json]: I guess it should have `^`.
```suggestion
    "lit": "^2.0.0"
```

**Review by Lodin — CHANGES_REQUESTED** [review]

File: frontend/packages/common-frontend/package.json (hunk 1 above)
Comment (Lodin) [reply frontend/packages/common-frontend/package.json]: Probably, it would be even better to use a peer dependency here.

Comment (haijian-vaadin) [reply frontend/packages/common-frontend/package.json]: done

File: frontend/packages/common-frontend/package.json [hunk 2]
```diff
@@ -45,7 +45,7 @@
     "access": "public"
   },
   "dependencies": {
```
Comment (Lodin) [inline frontend/packages/common-frontend/package.json]: Changing to peer dependencies gives the version choice to the user (or Flow framework). It guarantees that we won't have any dupes.
```suggestion
  "peerDependencies": {
```

**Review by Lodin — CHANGES_REQUESTED** [review]

**Review by Lodin — APPROVED** [review]
LGTM

**Review by Haprog — APPROVED** [review]

**Pull request merged**